gradle/root/coverage.gradle

```diff
@@ -27,8 +39,9 @@ task mockitoCoverage(type: JacocoReport, dependsOn: "test") {
     }
 
     doLast {
-        println "Jacoco report for client created: file://${reports.xml.destination.toURI().path}"
+        logger.lifecycle "Jacoco report for client created: file://${reports.xml.destination.toURI().path}"
+        logger.lifecycle "Jacoco HTML created: file://${new File(reports.html.destination, "index.html").toURI().path}"
```

Comment:
Does it actually generate html and xml? In the past the task could only generate only one kind of report.

Reply:
Yes, it is (it's enabled in the configuration)
.
For me the HTML link is enough. I don't know why the the link do XML was introduced. @TimvdLippe do you remember?

Reply:
That is used by CodeCov afaik

Reply:
Yes, but do they use also a link? :)

Reply:
Oh in that manner. I doubt it, but does it harm by reporting it?